core/test/element_unary_operations_test.cpp

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+#include <gtest/gtest.h>
+
+#include "scipp/core/value_and_variance.h"
+#include "scipp/units/unit.h"
+
+#include "../element_unary_operations.h"
+
+using namespace scipp;
+using namespace scipp::core;
+
+TEST(ElementSqrtTest, unit) {
+  const units::Unit m2(units::m * units::m);
+  EXPECT_EQ(element::sqrt(m2), units::sqrt(m2));
+}
+
+TEST(ElementSqrtTest, value) {
+  EXPECT_EQ(element::sqrt(1.23), std::sqrt(1.23));
```

Comment:
Maybe, to make sure we always cover all the `out` cases as well, a useful convention would be to have the same number of tests for without `out` and with `out`?
Meaning, this test should be split into `TEST(ElementSqrtTest, value_double)` and `TEST(ElementSqrtTest, value_float)`

Reply:
I don't think such a convention would work once we support more cases for the `out` argument. For example, as discussed at the dev. meeting, we may want to support `out` arguments with a different `dtype`, so there would be, e.g., 2 cases for normal operation but 4 cases with out arg (two extract cases switching `float <-> double`).

Reply:
yes that's a good point